src/TreeMenu.js

```diff
@@ -103,17 +103,26 @@ const Menu = (props) => {
         resource.options.hasOwnProperty('menuParent') &&
         resource.options.menuParent == parentResource.name
     );
+    const geResourceName = (slug) => {
```

Comment:
@sivaavkd is this intentional?

Reply:
you meant the typo ? sure, will correct in the next release :-)